src/Form/CaptchaPointForm.php

```diff
@@ -35,6 +39,7 @@ public function form(array $form, FormStateInterface $form_state) {
       '#type' => 'textfield',
       '#title' => $this->t('Form ID'),
       '#default_value' => $captcha_point->label(),
+      '#required' => TRUE,
```

Comment:
Can we extend one of the tests to make sure that the fields are required? Submit without them, assert the error messages. Just put it in one of the existing tests where we submit the form.